addons/mail/static/src/components/chatter_container/chatter_container.js

```diff
@@ -8,7 +8,7 @@ import { getMessagingComponent } from "@mail/utils/messaging_component";
 
 const { Component, onWillDestroy, onWillUpdateProps } = owl;
 
-const getChatterNextTemporaryId = (function () {
+export const getChatterNextTemporaryId = (function () {
```

Comment:
Ok as fix (minimal code change while fixing the issue)

Not to change in this PR: this pattern of allocating ids with `getSomethingId` functions should be replaced by proper fields in models instead.